### PR TITLE
metashell 5.0.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created by `brew bump`

---

Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  CMake Error at /home/linuxbrew/.linuxbrew/Cellar/cmake/3.27.1/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
    Could NOT find Python3 (missing: Python3_EXECUTABLE Interpreter) (Required
    is at least version "3.6")
```

**Comment by iMichka** [comment]
Let's try with the change I made previously: #128202
We were waiting for this release to see if we can build a Ventura bottle too.

**Comment by chenrui333** [comment]
> make: *** No rule to make target `templight'.  Stop.

**Comment by chenrui333** [comment]
linux test failure.

**Comment by chenrui333** [comment]
linux test failure.

**Comment by iMichka** [comment]
> ==> /home/linuxbrew/.linuxbrew/Cellar/metashell/5.0.0/bin/metashell -H
  Error: Clang binary not found.
  Error: metashell: failed
  An exception occurred within a child process:
    Minitest::Assertion: Expected /const\ int/ to match "clang++ not found. 

I think we are almost there to get this shipped. @sabel83 might have an idea.
Looks like it's not finding clang on Linux. I was under the impression that I installed its own clang? Or is clang provided by homebrew maybe now necessary at runtime?

File: Formula/metashell.rb [hunk 1]
```diff
@@ -19,26 +19,25 @@ class Metashell < Formula
...
 
   def install
-    ENV.cxx11
-
     # Build internal Clang
-    mkdir "3rd/templight/build" do
-      system "cmake", "../llvm", "-DLLVM_ENABLE_TERMINFO=OFF", *std_cmake_args
-      system "make", "templight"
-    end
+    system "cmake", "-S", "3rd/templight/llvm", "-B", "llvm_build", "-DLLVM_ENABLE_TERMINFO=OFF", *std_cmake_args
```
Comment (sabel83) [inline Formula/metashell.rb:34]: This is changing the build directory of templight (the clang build shipped with metashell). Metashell's own build scripts are sensitive to the location of the templight builds, which is likely to be the reason for the error you get from the CI. Take a look at `CMakeLists.txt` in the root of Metashell's source tree and `cmake/Modules/FindTemplight.cmake` for more details.

**Comment by sabel83** [comment]
> > ==> /home/linuxbrew/.linuxbrew/Cellar/metashell/5.0.0/bin/metashell -H
> > Error: Clang binary not found.
> > Error: metashell: failed
> > An exception occurred within a child process:
> > Minitest::Assertion: Expected /const\ int/ to match "clang++ not found.
> 
> I think we are almost there to get this shipped. @sabel83 might have an idea. Looks like it's not finding clang on Linux. I was under the impression that I installed its own clang? Or is clang provided by homebrew maybe now necessary at runtime?

Metashell comes with a special clang, you don't need to install another one for it. Take a look at my comment about the build location.

**Comment by iMichka** [comment]
Follow up PR: #140453
Found the issue, we forgot "-DLLVM_ENABLE_PROJECTS=clang"

**Pull request closed without merging**